hnn_core/cells_default.py

```diff
@@ -182,10 +187,7 @@ def basket(pos, cell_name='L2Basket', gid=None):
     for sec_name in p_secs:
         p_secs[sec_name]['sec_pts'] = sec_pts[sec_name]
 
-    cell = Cell(cell_name, pos=pos, gid=gid)
-    cell.build(p_secs, p_syn, topology, sect_loc)
-
-    return cell
+    return Cell(cell_name, pos, p_secs, p_syn, topology, sect_loc, gid)
 
 
 def pyramidal(pos, cell_name, override_params=None, gid=None):
```

Comment:
Similar to comment on basket cell, I am unsure what the preferred behavior is for `cell_name`. I think they should both have defaults or neither.